### ISSUE TITLE
Screen reader does not provide information of relation between table header and table data (DetailedList)

### ISSUE THREAD

**Issue by sereymonyros** [microsoft/fluentui#13348]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out if relevant)

#### Actual behavior:

Screen reader does not provide information of relation between table header and table data. Screen reader only reads table data but it does not announce that data is related to which table header.
Also screen reader does not pass sorting information on header control.

#### Expected behavior:

Screen reader should provide information of relation between table header and table data. Screen reader should announce like e.g. : 'pr1(table data) under Name(table header) and screen reader should be pass sorting( ascending or descending ) information on header control.

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by aneeshack4** [comment]
@sereymonyros Thanks for reaching out to us. You didn't provide a codepen so I ran Narrator on the DetailsList example on our docs site and it does provide the right information relating the table header and its data.... if you want it to read something else you can change the aria-labels when consuming this component. The screenreader simply follows ARIA structure and we can't control what it does beyond following the ARIA spec which we do here.
![image](https://user-images.githubusercontent.com/4161791/83040626-1670e700-a005-11ea-9b4c-62f0cf752bb1.png)
